core/src/main/scala/akka/persistence/cassandra/healthcheck/CassandraHealthCheck.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.cassandra.healthcheck
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.pattern.{ ask, AskTimeoutException }
+import akka.persistence.Persistence
+import akka.persistence.cassandra.PluginSettings
+import akka.persistence.cassandra.journal.CassandraJournal.HealthCheckQuery
+import akka.util.Timeout
+
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+import scala.util.control.NonFatal
+
+class CassandraHealthCheck(system: ActorSystem) extends (() => Future[Boolean]) {
+
+  private[akka] val log = Logging.getLogger(system, getClass)
```

Comment:
just private?

Reply:
You're right. It was copy-paste, because I wanted to create logger according to  existing code style and then I forget to change it to just private.